.travis.yml

```diff
@@ -5,30 +5,68 @@ matrix:
         # Run as a regular user
         - os: linux
           python: 2.7
+          env:
+            - SCAPY_COVERAGE=yes
+
+        - os: linux
+          python: 3.3
+          env:
+            - SCAPY_COVERAGE=yes
```

Comment:
`coverage` tend to make the build times way longer. I will suggest doing the same as today: only use `coverage` for root-based tests. A better option will me to use `coverage` for root-based Python 3.6 only tests.

Reply:
I wonder if its related to Python 3.x tests but the coverage percentage seemed to be lowered by ~ 3% with this PR.

Reply:
Shall we also add a `pypy` based Python 3 build ? It is the `pypy3` keyword.

A good approach will be to do that in a different PR when #861 is merged.

Reply:
I think 1.5 to 2 minutes for each non-root test with coverage is fine. Also non-root tests help cover some parts of the code not often hit (since Scapy is often used with root privileges) so it might be good to use coverage on it. What do you think?